### DISCUSSION TITLE
Erreur accessibilité exemple fichier d'attaque

### DISCUSSION THREAD

**Discussion by mxtla** [danymat/INSAnonym-utils#74] [Q&A, Bug report & Support] [answered]
Bonjour, 

Dans le README.md, il y a un "exemple de fichier d'attaque". Cependant, lorsqu'on clique sur le lien, on obtient un message de Google Docs comme quoi le document n'est pas public.

Serait-il possible d'y avoir accès ? 

Merci

**Comment by danymat** [comment]
Oui je le fais début de soirée. En attendant vous avez un exemple dans le guide utilisateur ou tout est renseigné pour la phase d'attaque

**Comment by danymat** [comment, marked as answer]
Rebonjour, le lien à été modifié sur un vrai exemple
